contents/docs/feature-flags/best-practices.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Feature flags best practices
```

Comment:
As suggested in my comment, if we're keeping both pages, we should change this to something else – e.g. "Feature flag tips".

Reply:
I'll drop "feature flags" from this title since its redundant (we're already in the feature flag docs). It'll affect cmd+k search, but I think its okay since I dont expect much cmd+k search for this